packages/@aws-cdk/aws-ecs-patterns/test/ec2/integ.scheduled-ecs-task.lit.ts

```diff
@@ -26,7 +27,7 @@ class EventStack extends cdk.Stack {
       memoryLimitMiB: 512,
       cpu: 1,
       environment: { name: 'TRIGGER', value: 'CloudWatch Events' },
-      scheduleExpression: 'rate(1 minute)'
+      schedule: events.Schedule.fromRate(1, events.TimeUnit.Minute),
```

Comment:
I don’t like the “from” prefix. It doesn’t read right. It’s not “schedule from rate”. I think just “Schedule.rate” feels better.

Reply:
Could we maybe use the `Duration` class here somehow? This is a length of time...

Reply:
It's not merged yet, is it?